### DISCUSSION TITLE
Does Xonsh VI mode have clipboard buffers

### DISCUSSION THREAD

**Discussion by Spitfire1900** [xonsh/xonsh#5678] [Q&A]
I frequently with copy a string from an IDE or web browser to paste it into a Xonsh session in VI mode. I'll use normal mode's `c+w` or `d+t+<char>` prior to pasting. This overwrites the clipboard buffer.

is there a good approach to this?

**Comment by anki-code** [comment]
VI mode is the part of prompt-toolkit
